### PR TITLE
feat: Add Light/Dark Toggle to samples

### PR DESCRIPTION
**Pull request by kazo0**
## PR Type

What kind of change does this PR introduce?

- Feature

## Description

Adds a Light/Dark theme toggle to the samples

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested UWP
- [ ] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		/// </summary>
+		public static ApplicationTheme GetSystemApplicationTheme()
+		{
+#if __ANDROID__
+			if ((int)Android.OS.Build.VERSION.SdkInt >= 28)
+			{
+				var uiModeFlags = Android.App.Application.Context.Resources.Configuration.UiMode & Android.Content.Res.UiMode.NightMask;
+				if (uiModeFlags == Android.Content.Res.UiMode.NightYes)
+				{
+					return ApplicationTheme.Dark;
```
Comment (jeromelaban) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs:23]: Don't we have this in Uno ? @davidjohnoliver

Comment (kazo0) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs:23]: We might, I just copied it quickly from the Uno.Gallery repo. I will check

Comment (kazo0) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs:23]: I don't see it anywhere

Comment (davidjohnoliver) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Helpers/SystemThemeHelper.cs:23]: It's `internal` in Uno.

We may be able to use the `ThemeListener` class from WCT, now that @MartinZikmund has implemented settings for `UISettings`. (See also https://github.com/unoplatform/nventive-private/issues/38)

**Review by carldebilly — APPROVED** [review]

**Pull request merged**